examples/benchmark_xgboost.py

```diff
@@ -0,0 +1,1100 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct 18 15:53:27 2016
+
```

Comment:
Is this script the same as `benchmark.py`? I personally prefer to keep only one of them.